apps/system/js/captive_portal.js

```diff
@@ -51,22 +53,22 @@ var CaptivePortal = {
       return;
     }
 
-    this.captiveNotification_onClick = (function() {
-      this.notification.removeEventListener('click',
-                                            this.captiveNotification_onClick);
+    this.captiveNotification_onClick = notification => {
+      notification.removeEventListener('click',
```

Comment:
I think this should be notification.removeEventListener(this.captiveNotification_onClick) - so you remove the function reference, not the result of a call to it